jobclass/rebuild-rename.rb

```diff
@@ -45,8 +45,8 @@
       # RENAME
       task.transaction {
         task.create_dummy_table dest_table
-        task.rename_table dest_table, prev_table
-        task.rename_table work_table, dest_table
+        task.rename_table dest_table, prev_table.name
+        task.rename_table work_table, dest_table.name
```

Comment:
👈  [check] pass only table name to rename_table.  This line execute a desired alter statement.
https://github.com/bricolages/bricolage/blob/a53284f0db8f795fb1a51aff8729cd53faccda5b/lib/bricolage/psqldatasource.rb#L319-L321